rubin_sim/scheduler/utils/sky_area.py

```diff
@@ -575,7 +576,8 @@ def combine_maps(self, trim_overlap=True):
             self.total += self.total_perfilter[f]
 
     def return_maps(self):
-        """Call combine_maps and return self.total and self.total_perfilter."""
+        """Call combine_maps and return self.total and self.total_perfilter.
+
+        XXX: need to document the output here. """
```

Comment:
Replace XXX: need to document the output here with 

```
Returns
--------
self.total, self.total_perfilter : `np.ndarray`, `np.ndarray`
    HEALPix maps reflecting the total expected number of visits per pointing, and the number of visits per pointing per filter. These can then be scaled appropriately into target maps for the scheduler. 
```

(actually, I'd suggest even doing that scaling here before returning them?)